lib/fluent/plugin/in_cloudwatch_ingest.rb

```diff
@@ -279,56 +284,60 @@ def run
         begin
           state = State.new(@state_file_name, log)
         rescue => boom
-          log.info("Failed lock state. Sleeping for #{@error_interval}: "\
-                   "#{boom.inspect}")
+          log.info("Failed lock state. Sleeping for #{@error_interval}: #{boom.inspect}")
           sleep @error_interval
           next
         end
 
         event_count = 0
-
-        # Fetch the streams for each log group
+        # For each log group
         log_groups(@log_group_name_prefix).each do |group|
-          # For each log stream get and emit the events
-          log_streams(group, @log_stream_name_prefix).each do |stream|
-            state.store[group][stream] = {} unless state.store[group][stream]
-
-            log.info("processing stream: #{stream}")
-
-            # See if we have some stored state for this group and stream.
-            # If we have then use the stored forward_token to pick up
-            # from that point. Otherwise start from the start.
-
-            begin
-              event_count += process_stream(group, stream,
-                                            state.store[group][stream]['token'],
-                                            @event_start_time, state)
-            rescue Aws::CloudWatchLogs::Errors::InvalidParameterException
-              metric(:increment, 'api.calls.getlogevents.invalid_token')
-              log.error('cloudwatch token is expired or broken. '\
-                        'trying with timestamp.')
-
-              # try again with timestamp instead of forward token
-              begin
-                timestamp = state.store[group][stream]['timestamp']
-                timestamp = @event_start_time unless timestamp
+          begin
```

Comment:
this is a new "do-while" block, so things have been right idented.. see line 339